README.md

```diff
@@ -11,6 +11,15 @@ at the heart of the [Wikibase software](http://wikiba.se/).
 ### 3.0.0 (alpha)
 
 * Removed cloning from `MultiTerm.getTexts`.
+* Turned EntityId into a simple wrapper around an opaque serialization string
+  * Removed EntityId::getNumericId
+  * Removed EntityId::getEntityType
+  * Removed EntityId::getPrefixedId
+  * Introduced EntityId::getSerialization
```

Comment:
That would be a static class method. This is about a instance methods. You should either do `::` or `.prototype`.